tests/files/.gitignore

```diff
@@ -1,2 +1,3 @@
 *
-!.gitignore
\ No newline at end of file
+!.gitignore
+!database.sqlite
```

Comment:
I've added a simple sample sqlite to verify that the dump with a test. If this isn't necessary then the sample db and the added test can be removed. The previous tests and code just check for the necessary command being generated and never actually validated if it worked.